docs/api/en/renderers/shaders/UniformsLib.html

```diff
@@ -12,6 +12,38 @@ <h1>[name]</h1>
 		<p class="desc">Uniforms library for shared webgl shaders</p>
 
 
+		<h2>A Note on Usage</h2>
```

Comment:
This page is the wrong place for documenting the restrictions of `UniformUtils.clone()`. Please do it here:

https://threejs.org/docs/index.html#api/en/renderers/shaders/UniformsUtils

Reply:
I've removed the new section so the PR can be merged. The docs can be updated with a different PR.